modules/ROOT/pages/release-notes.adoc

```diff
@@ -31,30 +31,35 @@ include::_attributesLocal.adoc[]
 *** `deprecated.shadow.doc_id_regex`
 *** `deprecated.shadow.feed_type`
 
-* New options
+* New and Changed options
 ** `db.import_partitions` - Controls the number of partitions for import sharding (see xref:config-properties.adoc#databases-foo_db-import_partitions[databases.$db.import_partitions])
 // `unsupported.use_stdlib_json` - Couchbase support-use if required. EE-only, Default: false. Uses Go's stdlib JSON library, instead of a faster third-party JSON library
 
-* Changed Options
-
 ** The default behavior of `db.import_docs` has changed (see xref:config-properties.adoc#databases-foo_db-import_docs[databases.$db.import_docs]).
 +
 This will provide the flexibility for "import heavy" use case users to be able to isolate the import nodes from the internet/client-facing nodes
 
+*Enhancements*
+
+{jira-url}/CBG-584[*CBG-584*] SyncGateway now supports scenarios where alternate addresses are provided on Couchbase Server.
+This change adds overrides for `ConnectBucket` and `Connect` to remap _hostnames_ based on `AlternateAddress`.
```

Comment:
Not sure this is nessesary to document - it's an internal implementation detail which may change.

```suggestion
```

Reply:
@bbrks change accepted, please approve.